deploy/injector-mutating-webhook.yaml

```
@@ -22,10 +22,10 @@ webhooks:
         namespace: "vault"
       caBundle: ""
     rules:
-      - operations: ["CREATE", "UPDATE"]
+      - operations: ["CREATE"]
         apiGroups: [""]
         apiVersions: ["v1"]
-        resources: ["deployments", "jobs", "pods", "statefulsets"]
+        resources: ["pods"]
```

Would you mind adding `scope` here too? This was [recently added](https://github.com/hashicorp/vault-helm/pull/1037) to our helm chart and it would good to be consistent.

```suggestion
        resources: ["pods"]
        scope: "Namespaced"
```

---

Applied!